dj_rest_auth/models.py

```diff
@@ -1,5 +1,21 @@
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.utils.module_loading import import_string
 
+default = 'rest_framework.authtoken.models.Token'
 
-TokenModel = import_string(getattr(settings, 'REST_AUTH_TOKEN_MODEL', 'rest_framework.authtoken.models.Token'))
```

Comment:
This will be evaluated only once at import time, so changing REST_AUTH_TOKEN_MODEL in the test suite will have no effect. To enable testing a different value for this option, we have to define a callable to get the token model instead.

Reply:
https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.modify_settings

> Warning
> 
> The settings file contains some settings that are only consulted during initialization of Django internals. If you change them with override_settings, the setting is changed if you access it via the django.conf.settings module, however, Django’s internals access it differently. Effectively, using override_settings() or modify_settings() with these settings is probably not going to do what you expect it to do.